src/sql/workbench/api/node/sqlExtHost.api.impl.ts

```diff
@@ -121,8 +121,8 @@ export function createApiFactory(
 				getUriForConnection(connectionId: string): Thenable<string> {
 					return extHostConnectionManagement.$getUriForConnection(connectionId);
 				},
-				connect(connectionProfile: azdata.IConnectionProfile): Thenable<azdata.ConnectionResult> {
-					return extHostConnectionManagement.$connect(connectionProfile);
+				connect(connectionProfile: azdata.IConnectionProfile, saveConnection: boolean, showDashboard: boolean): Thenable<azdata.ConnectionResult> {
```

Comment:
are these changes back-compat?  e.g., does postgres still work, etc.

Reply:
It'd be good to make these be optional

---
In reply to: [289215111](https://github.com/microsoft/azuredatastudio/pull/5542#discussion_r289215111) [](ancestors = 289215111)

Reply:
I'm fixing a bug where the API wasn't updated properly. I'm not adding this.